backend/invoke_ai_web_server.py

```diff
@@ -81,7 +81,13 @@ def setup_flask(self):
         }
 
         if opt.cors:
-            socketio_args["cors_allowed_origins"] = opt.cors
+            _cors = opt.cors
+            if hasattr(_cors, "__len__"):
```

Comment:
What's this for? If you're testing for string, use `isinstance(opt.cors, str)` (or `list`, `dict`, etc...) https://docs.python.org/3.10/library/functions.html#isinstance

Otherwise, *anything* with a 'len' attribute will pass here

Reply:
I was attempting to check to see if it is an array. I'm not a python developer but it sounds like this test would work regardless of whether it is an array or a string? I'll edit that to use the correct invocation as you noted.